src/lib/swapper/swappers/CowSwapper/cowExecuteTrade/cowExecuteTrade.test.ts

```diff
@@ -57,7 +42,34 @@ jest.mock('../utils/helpers/helpers', () => {
   }
 })
 
-const ethereumMock = jest.mocked(ethereum)
+const actualChainAdapters = jest.requireActual('@shapeshiftoss/chain-adapters')
+
+const mockEthereum = {
+  getBIP44Params: jest.fn(() => actualChainAdapters.ethereum.ChainAdapter.defaultBIP44Params),
+  getChainId: () => KnownChainIds.EthereumMainnet,
+  signMessage: jest.fn(() => Promise.resolve(Signature)),
+} as unknown as ethereum.ChainAdapter
+
+const mockGnosis = {
+  getBIP44Params: jest.fn(() => actualChainAdapters.gnosis.ChainAdapter.defaultBIP44Params),
+  getChainId: () => KnownChainIds.GnosisMainnet,
+  signMessage: jest.fn(() => Promise.resolve(Signature)),
+} as unknown as ethereum.ChainAdapter // wrong type but doesn't matter here
```

Comment:
Since we're mocking the relevant methods here, agree don't need to do test gymnastics to make the `gnosis.ChainAdapter` work with the missing `validateEnsAddress` but I'd either add a comment about the two chain-adapters having slightly different types but being mocked so same thing at runtime in the context of these tests, or even better, use `EVMChainAdapter`, which won't produce such error, see:

<img width="863" alt="image" src="https://github.com/shapeshift/web/assets/17035424/4a084fe1-9d8e-4cfa-9696-f5ed7a42de7b">
<img width="893" alt="image" src="https://github.com/shapeshift/web/assets/17035424/77062ccd-9ff8-4d54-b77f-1bad866365f4">

Reply:
https://github.com/shapeshift/web/pull/4595/commits/0cc0dddcddf5cfa71691efc9782e74a74d34de92